addons/web/static/tests/views/list_tests.js

```diff
@@ -1501,6 +1501,119 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('editable list: headers widths are only frozen on any interraction if the list is visible', async function (assert) {
+        assert.expect(8);
+
+        this.data.foo = {
+            fields: {
+                bar: { string: "Bar", type: "boolean" },
+                o2m1: { string: "O2M field", type: "one2many", relation: "bar" },
+                o2m2: { string: "O2M field", type: "one2many", relation: "bar" },
+            },
+            records: [{ id: 1, bar: true, o2m1: [1], o2m2: [1] }],
+        };
+        this.data.bar = {
+            fields: {
+                titi: { string: "Small char", type: "char", sortable: true },
+                grosminet: { string: "Beeg char", type: "char", sortable: true },
+            },
+            records: [
+                {
+                    id: 1,
+                    titi: "Tiny text",
+                    grosminet:
+                        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ' +
+                        'Ut at nisi congue, facilisis neque nec, pulvinar nunc. ' +
+                        'Vivamus ac lectus velit.',
+                },
+            ],
+        };
+        var form = await createView({
+            View: FormView,
+            model: 'foo',
+            data: this.data,
+            res_id: 1,
+            viewOptions: { mode: 'edit' },
+            arch: '<form>' +
+                    '<sheet>' +
+                        '<field name="bar"/>' +
+                        '<notebook>' +
+                            '<page string="Bar1">' +
+                                '<field name="o2m1">' +
+                                    '<tree editable="bottom">' +
+                                        '<field name="titi"/>' +
+                                        '<field name="grosminet"/>' +
+                                        '<button icon="fa-heart"/>' +
+                                    '</tree>' +
+                                '</field>' +
+                            '</page>' +
+                            '<page string="Bar2" attrs="{\'invisible\': [(\'bar\', \'=\', True)]}">' +
```

Comment:
why invisible modifier? I think you mixed the two tests we we talking about:
1) notebook, 2 pages, a one2many in the second page
2) a boolean and a one2many with a modifier (invisible at first)

No need for two one2many in any of those tests (keep it as simple as possible)

Reply:
Actually I don't think we need a test for the invisible modifier right now, since it works exactly the same way as a hidden page (display: none). We might need to extend this test if we change the way invisible fields work though.

Reply:
Well there are the two scenarios that came in our minds when we considered invisible list views... As we thought about it, I think it would be nice to have a test so that if we change it in the future, we ensure that we won't break this scenario. Moreover, it's almost written already.